source/scripts/data/monsters.js

```diff
@@ -36,6 +36,13 @@ export default {
         color: DATA.COLORS.WHITE,
         health: 5,
         strength: 5,
+        turnCounter: function() {
+            this.somecount = this.somecount + 1 || 0
+            if(this.somecount % 4 == 0) {
+                return this.phase = true
+            }
+            return this.phase = false
```

Comment:
It'll be easier (in my opinion) to read if this `if` statement just sets `this.phase`, and then the function has a single `return this.phase` at the bottom.